src/httpserver/details/http_endpoint.hpp

```diff
@@ -120,12 +120,12 @@ class http_endpoint
             return chunk_positions;
         }
 
-        const bool is_family_url() const
+        const bool is_family_url()
         {
             return family_url;
         }
 
-        const bool is_regex_compiled() const
+        const bool is_regex_compiled()
```

Comment:
what was the issue with the is_family_url and is_regex_compiled methods being const?